lib/v2/jsu/router.js

```diff
@@ -0,0 +1,8 @@
+module.exports = (router) => {
+    // 创新中心
+    router.get('/cxzx/:types?', require('./cxzx'));
+    router.get('/rjxy', require('./rjxy'));
+    router.get('/stxy', require('./math'));
+    router.get('/notice', require('./universityindex'));
+    router.get('/jwc/:types?', require('./jwc'));
```

Comment:
Please insert entries in alphabetical order.
```suggestion
    router.get('/cxzx/:types?', require('./cxzx'));
    router.get('/jwc/:types?', require('./jwc'));
    router.get('/notice', require('./universityindex'));
    router.get('/rjxy', require('./rjxy'));
    router.get('/stxy', require('./math'));
```